src/Illuminate/Auth/Access/Gate.php

```diff
@@ -398,7 +398,7 @@ public function getPolicyFor($class)
         }
 
         if (! is_string($class)) {
-            return null;
+            return;
```

Comment:
But `return;` !== null its void, isn't it?

Reply:
Which is interpreted as null, thus many different return type - as return docblock suggests

Reply:
Thats all about PHP weak types :(

Reply:
You'll notice that this is the case with all other `return null;` / `(void)`
This PR just enforces consistency.